README.rst

```diff
@@ -6,21 +6,17 @@
    :target: https://docs.ansible.com/ansible-lint/rules/default_rules.html
    :alt: Ansible-lint rules explanation
 
-.. image:: https://img.shields.io/badge/Code%20of%20Conduct-Ansible-silver.svg
+.. image:: https://img.shields.io/badge/Code%20of%20Conduct-silver.svg
```

Comment:
I don't think this is correct. It breaks the badge and also I wanted to indicate that it's Ansible's CoC, not a separate one. FWIW this is up to @gundalow to decide.

Reply:
This works fine and is already used by other projects, see https://pypi.org/project/molecule/ -- I think that I asked @gundalow at that time but we can definitely wait for his input.

In the end, what matters is to have a link towards the CoC document, where it is listed and how it is formatted is more of an implementation detail.

Reply:
I'm OK with this

Reply:
Then maybe consider changing the color scheme. White text on silver is barely visible. Before, it was just one word and now it's the important phrase that becomes "invisible".

Reply:
Well, is the same color it used to be for Ansible. Still, I made it black to avoid lack of contrast.